contrib/depends/packages/ncurses.mk

```diff
@@ -35,9 +36,8 @@ define $(package)_set_vars
   $(pacakge)_config_opts+=--disable-symlinks
   $(pacakge)_config_opts+=--enable-warnings
   $(pacakge)_config_opts+=--enable-assertions
-  $(pacakge)_config_opts+=--disable-home-terminfo
-  $(package)_config_opts+=--with-default-terminfo-dir=/etc/terminfo
-  $(package)_config_opts+=--with-terminfo-dirs="/etc/terminfo:/lib/terminfo:/usr/share/terminfo"
+  $(package)_config_opts+=--with-default-terminfo-dir=/etc/_terminfo_
+  $(package)_config_opts+=--with-terminfo-dirs=/etc/_terminfo_
```

Comment:
I guess that means "mind yours" ?

Reply:
We want to avoid using any system-installed terminfo DB because if the system-installed libterminfo was configured with different options than ours, the DB files are incompatible. So we pick a path that is unlikely to actually be in use.